status-im/status-react
# [#6718] Wallet onboarding setup re-style for new wallet flow

fixes #6718  Wallet onboarding setup re-style for new wallet flow

### Summary:

This re-styles the wallet onboarding screens according to for `wallet-01` in the following figma https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction.

### Testing notes:

Check that the screens match the figma and that they function as intended.

### Steps to test:
- Open Status
- Create a new account
- Nav to wallet
- Click "Let's get setup"
- compare the screen to `wallet-01` on the figma linked above
- Click "Got it"
- compare the content of the alert dialog to the alert in `wallet-01`

Check the modal view as well:
- Ensure in Development Mode
- add the Status Test Dapp
- Open Status Test Dap
- Click "Transactions"
- Click "Send one Tx in batch"
- the setup modal should now be visible and should look correct

Check that once the signing phrase accepted that the onboarding screens don't show up when you return to the wallet.

Thanks!


status: ready


<blockquote><img src="/file/SIeZPHvrX7KUwEBW8VQct8C5/thumbnail?ver=thumbnails/57a1f597-484c-4f1c-a513-79b19f7c3b3a" width="48" align="right"><div><img src="https://static.figma.com/app/icon/1/favicon.svg" height="14"> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>
